pkg/context/context.go

```diff
@@ -117,35 +120,20 @@ func (c *Context) CopyFile(src, dest string) error {
 	return errors.WithStack(err)
 }
 
-// WriteOutput writes the given lines to a file in the
-// output directory
-func (c *Context) WriteOutput(lines []string) error {
-	exists, err := c.FileSystem.DirExists("/cnab/app/porter/outputs")
+// WriteOutput writes the provided bytes to a file named by the provided filename
+// to Porter's outputs directory
+func (c *Context) WriteOutput(filename string, bytes []byte) error {
```

Comment:
I was just reviewing the mixin code for your porter-azure PR and realized that this function name is really confusing. When people see `p.Context.WriteOutput` it could easily be construed as "write output/logs to stdout". Which is my fault.

If you agree that the name is really confusing, now would be the best time to fix it. What do you think? Maybe we could change it to `WriteMixinOutputToFile` or something much more explicit.

Reply:
Sounds good to me; I'll update this name here and in the mixin PRs to be `WriteMixinOutputToFile`